bin/octokit-payloads.js

```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env node
+
+const fs = require("fs");
+const { ajv, validate } = require("../payload-schemas");
+
+let hasErrors = false;
+const payloads = `./payload-examples/api.github.com`;
+
+fs.readdirSync(payloads)
+  .filter((filename) => filename.endsWith(".json"))
+  .forEach((filename) => {
+    const [event, ...unused] = filename.split(".");
```

Comment:
No need for the `unused` part

```suggestion
    const [event] = filename.split(".");
```